examples/bert_style_qa/handler.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+import sys
+
+import torch
+from transformers import (AutoConfig, AutoTokenizer,
+                          AutoModelForQuestionAnswering)
+
+from dynalab.handler.base_handler import BaseDynaHandler
+from dynalab.tasks.qa import TaskIO
+
+sys.path.append("/home/model-server/code")
```

Comment:
Please can you init a new repo and start from that template handler please? We've wrapped this path in the `ROOTPATH` variable.

Reply:
Let me know if the updated version is ok, or you had something else in mind?